.github/ci-scripts/bash/functions-for-build

```diff
@@ -488,6 +488,28 @@ python_docker_build_and_push() {
   set +e
 }
 
+python_setup_build() {
+  if [ -f requirements.txt ]; then 
+    python3 -m pip install -r requirements.txt --dry-run --trusted-host ${NEXUS_SERVER_ENDPOINT}; 
+  fi
```

Comment:
move function `installPythonDependencies` from build file to script and use that for installing dependencies